ui/eureka/eureka.js

```diff
@@ -1900,7 +1900,7 @@ let Options = {
           },
           trackerName: {
             en: 'Barong',
-            de: 'Baron',
+            de: 'Barong',
```

Comment:
Double checked this one, and you're right that it is Barong.  Thanks!

Reply:
arf i correct

Reply:
The Name of the Mob is "Baron Feuermähne" and not Barong in german